### ISSUE TITLE
dots in object mapping ignored

### ISSUE THREAD

**Issue by visat1** [strapdata/elassandra#295]
**Describe the feature**: Dots in object-field names are silently ignored
**Elassandra version**: 6.2.3.15
**Plugins installed**: []
**JVM version** (`java -version`):
OpenJDK Runtime Environment (build 1.8.0_212-8u212-b01-1~deb9u1-b01)
OpenJDK 64-Bit Server VM (build 25.212-b01, mixed mode)

**OS version** (`uname -a` if on a Unix-like system):
Linux db1-dev 4.9.0-9-amd64 #1 SMP Debian 4.9.168-1+deb9u2 (2019-05-13) x86_64 GNU/Linux

**Description of the problem including expected versus actual behavior**:

**Steps to reproduce**:
 **1. Create empty index - e.g. testindex**
 **2. PUT /testindex/_doc/1**
Request-Body:
{
    "us.er" : "test",
    "postdate" : "2009-11-15T14:12:12",
    "message" : "Test123"
}

i get the following response document:
{
    "_index": "testindex",
    "_type": "_doc",
    "_id": "1",
    "_version": 1,
    "result": "created",
    "_shards": {
        "total": 1,
        "successful": 1,
        "failed": 0
    }
}

 **3. GET on http://127.0.0.1:9200/testindex/_doc/1**
{
    "_index": "testindex",
    "_type": "_doc",
    "_id": "1",
    "_version": 1,
    "found": true,
    "_source": {
        "postdate": "2009-11-15T14:12:12.000Z",
        "message": "Test123"
    }
}

the field with the dot in the name (us.er) is silently ignored.

**Comment by vroyer** [comment]
Yes, there is a bug when the source document is flattened. The current workaround is to insert document with nested fields like this:

```
{
"us": { "er" : "test" },
"postdate" : "2009-11-15T14:12:12",
"message" : "Test123"
}
```
This will be fixed in the next release by rebuilding the input doc as a tree.
Thanks for reporting this issue.

**Comment by vroyer** [comment]
Fixed in version 6.2.3.17
